frontend/src/App.js

```diff
@@ -102,7 +102,7 @@ function App() {
           </Route>
           <Route path="/stylists">
             <Sidebar
-              items={stylistSidebarRoutes(isActiveAppointment)}
+              items={stylistSidebarRoutes(isActiveAppointment, userRole==1)}
```

Comment:
What is the purpose of adding this logic here?

Reply:
Refer to comment below